sql/updates/world/master/2018_04_05_01_world_at_circular.sql

```diff
@@ -0,0 +1,17 @@
+DROP TABLE IF EXISTS `spell_areatrigger_circular`;
+CREATE TABLE `spell_areatrigger_circular`(
+    `SpellMiscId` INT(10) UNSIGNED NOT NULL,
+    `CircleRadius` FLOAT NOT NULL DEFAULT 0,
+    `BlendFromRadius` FLOAT NOT NULL DEFAULT 0,
+    `InitialAngle` FLOAT NOT NULL DEFAULT 0,
+    `ZOffset` FLOAT NOT NULL DEFAULT 0,
+    `CounterClockWise` TINYINT(3) NOT NULL DEFAULT 0,
+    `CanLoop` TINYINT(3) NOT NULL DEFAULT 0,
+    `VerifiedBuild` INT(10) NOT NULL DEFAULT 0,
+    PRIMARY KEY (`SpellMiscId`)
+);
+
+DELETE FROM `spell_areatrigger` WHERE SpellMiscId = 5016;
+INSERT INTO `spell_areatrigger` VALUES (5016, 10407, 0, 0, 0, 0, 26, 31636, 21000, 25996);
+
+INSERT INTO `spell_areatrigger_circular` VALUES (5016, 12.27271, 12.27271, 2.286397, 0, 0, 1, 25996);
```

Comment:
I would remove this line.. because its not really valid..

Info: this areatrigger has to be scripted because each at spawned got a different radius and initialAngle

Reply:
I'll let this as AT example if people want to test it, we'll just have to remove that when AT is scripted

Reply:
Codestyle-wise, it is recommended to include the column names to be populated. For the table `spell_areatrigger_circular` I guess you could keep it because it is already listed above, creating the table, but it ought to be added for the `spell_areatrigger` inserts.